indico/modules/events/timetable/controllers/manage.py

```diff
@@ -113,3 +114,7 @@ def _process_PATCH(self):
         if updates:
             update_timetable_entry(self.timetable_entry, updates)
         return jsonify()
+
+    def _process_DELETE(self):
+        """Delete a timetable entry"""
+        delete_timetable_entry(self.timetable_entry)
```

Comment:
adding session blocks is possible now.

Maybe it's better to only delete the timetable entry in here and send a DELETE request to `contributions.manage_contrib_rest` in case it's not a conference? Deleting a contribution automatically unschedules it so it'd be cleaner like that